app/src/ui/lib/spacial-constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+These constants are put in place to mirror the styles in our variables.scss
+so we have centralized place to update them if the css would change.
+
+NB - Preferably we should dynamically find heights and widths of html elements
+when possible to avoid the need to use static heights and widths.
+*/
+
+/** The app width for standard padding as defined in the 'variables.sccs' */
+export const spacing = 10
```

Comment:
The next step would be having _something_ sharing this directly with our CSS (some kind of variable replacement like what we do with `__DARWIN__` and such), but that's a bit more complicated.

Reply:
Yeah.. I thought about that, and also feel like that is a better use of time that if we get to the point where we find we actually need to change these to implement something more sophisticated and not over engineer for a what if that feels small right now.